packages/validator/src/validator.ts

```diff
@@ -159,6 +162,32 @@ export class Validator {
       this.clock.start(this.controller.signal);
       this.chainHeaderTracker.start(this.controller.signal);
 
+      const validatorKeys = this.validatorStore.getValidatorKeys();
+
+      for (const pubkeyHex in validatorKeys) {
+        let feeRecipient = "";
+
+        try {
+          feeRecipient = validatorStore.getFeeRecipient(pubkeyHex);
+        } catch (e) {
+          this.logger.warn("Cannot retrieve Fee Recipient", {}, e as Error);
```

Comment:
generally error messages use lower case consistently

```suggestion
          this.logger.warn("Cannot retrieve fee recipient", {}, e as Error);
```